app.js

```diff
@@ -58,6 +80,10 @@ app.get('/pieces-ready-to-be-sent', async function (req, res, next) {
     .send({ data });
 });
 
+app.post('/debug-resync-error-flows', async function (req, res, next) {
+  await syncFlowsByStatus([PARLIAMENT_FLOW_STATUSES.VP_ERROR]);
```

Comment:
**issue**: Endpoints need to handle the `res` object otherwise the request is never closed and it looks like it hangs to the client. Just a `return res.status(204).send();` would be enough here for now, I think.

Reply:
fixed in [8edb714](https://github.com/kanselarij-vlaanderen/vlaams-parlement-sync-service/pull/7/commits/8edb7144d5ca03ce7a2ca682235a771b7b75056e)